makefile

```diff
@@ -0,0 +1,12 @@
+all: proto qkchash
+
+proto: ./quarkchain/cluster/grpc_client.proto
```

Comment:
nit: can skip `./` as the default path should be the same as the Makefile itself

Reply:
Done.